### PR TITLE
Add new product logos and logotypes

### PR DESCRIPTION
**Pull request by domidomi**
https://www.figma.com/file/G7BPkTWmjWayauY6t8eeHf/Pencil-Foundations?node-id=514%3A54

### CODE REVIEW

File: src/images/logos/product/brainly-logo-math-solver.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 80 80" xmlns="http://www.w3.org/2000/svg">
```
Comment (coderitual) [inline src/images/logos/product/brainly-logo-math-solver.svg:1]: @domidomi the logo names are slightly different in the figma file and from where we left off yesterday. Could you please contact Olga and David to confirm what happened?

Comment (domidomi) [reply src/images/logos/product/brainly-logo-math-solver.svg:1]: @coderitual Olga is going to adjust filenames in Figma

Comment (domidomi) [reply src/images/logos/product/brainly-logo-math-solver.svg:1]: @coderitual seems that we should remove "brainly" from logos file names though!

Comment (coderitual) [reply src/images/logos/product/brainly-logo-math-solver.svg:1]: thanks for checking @domidomi

**Review by clxandstuff — APPROVED** [review]

**Review by coderitual — APPROVED** [review]

**Pull request merged**